issuance/message/schema/credential-object-schema.json

```
@@ -0,0 +1,71 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "CredentialObject",
+  "type": "object",
+  "properties": {
+    "@type": {
+      "type": "string"
+    },
+    "format": {
+      "type": "string"
+    },
+    "credentialType": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "bindingMethods": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "cryptographicSuites": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "issuancePolicy": {
+      "type": "object",
+      "properties": {
+        "permission": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "action": {
+                "type": "string"
+              },
+              "constraint": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "leftOperand": {
+                      "type": "string"
+                    },
+                    "operator": {
+                      "type": "string"
+                    },
+                    "rightOperand": {
+                      "type": "string"
+                    }
+                  },
+                  "required": ["leftOperand", "operator", "rightOperand"]
+                }
+              }
+            },
+            "required": ["action", "constraint"]
+          }
+        }
+      },
+      "required": ["permission"]
+    },
+    "offerReason": {
+      "type": "string"
+    }
+  },
+  "required": ["@type", "format", "credentialType"]
```

```suggestion
  "required": ["@context", "@type", "format", "credentialType"]
```
This should be mandatory everywhere - in all schemas (except those that define jwt tokens).

---

Done